.taprc

```diff
@@ -1,4 +1,4 @@
-statements: 90
-lines: 90
-functions: 78
-branches: 90
+statements: 100
```

Comment:
100 should be the default. try removing this file if there's no other option we're using

Reply:
I've removed the file, please re-approve. Thanks!